.emacs.d/lisp/code/editorconfig-c.el

```diff
@@ -0,0 +1,27 @@
+;; Enable EditorConfig mode
+(editorconfig-mode +1)
+
+(defun kotct/editorconfig--disable-smart-tabs-if-indent-tabs-mode-disabled (&optional hash)
+  "Disables `smart-tabs-mode' if `indent-tabs-mode' is non-truthy."
+  (if (not indent-tabs-mode)
+      (progn
+        (smart-tabs-mode -1))))
+
+(add-hook 'editorconfig-custom-hooks 'kotct/editorconfig--disable-smart-tabs-if-indent-tabs-mode-disabled)
+
+(defvar kotct/editorconfig--suppress-core-not-found-warning
+  nil
+  "If non-nil, warning on startup is suppressed.")
+
+(defun kotct/editorconfig-check-for-core (&optional suppress)
```

Comment:
Aye, but we don't want the functions mixed in with the ones that are interactive when you're using something like an autocompleter.